wren-ai-service/Makefile

```diff
@@ -12,6 +12,7 @@ dev-down:
 
 ## wren-ai-service related ##
 start:
+	cp .env.example .env
```

Comment:
I do believe we could remove the command and instead modify the function for loading the environment variables. What do you think?

Reply:
synced